main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/GoServer/newsapi"
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+func get(httpResponseWriter http.ResponseWriter, request *http.Request) {
+	httpResponseWriter.Header().Set("Content-Type", "application/json")
+	httpResponseWriter.WriteHeader(http.StatusOK)
+	httpResponseWriter.Write([]byte(`{"message": "get called"}`))
+}
+
+func post(httpResponseWriter http.ResponseWriter, request *http.Request) {
+	httpResponseWriter.Header().Set("Content-Type", "application/json")
+	httpResponseWriter.WriteHeader(http.StatusCreated)
+	httpResponseWriter.Write([]byte(`{"message": "post called"}`))
+}
+
+func put(httpResponseWriter http.ResponseWriter, request *http.Request) {
+	httpResponseWriter.Header().Set("Content-Type", "application/json")
```

Comment:
any chance we can avoid repeat code for this?

Reply:
Good call.
Addressed.